packages/alice/analysis_options.yaml

```diff
@@ -1,16 +1,10 @@
-1include: package:very_good_analysis/analysis_options.yaml
+analyzer:
```

Comment:
What is the reason of using `flutter_lints` instead of `very_good_analysis`?

Reply:
One is standard and always up to date, the other one is custom and outdated (8 months +). It also pointed to some potential issues like using contexts across async gaps, using `forEach` where a `for` loop should be used, `const` optimizations, etc.